cmd/v0.17/pingsource-cleanup/main.go

```diff
@@ -72,7 +72,8 @@ func main() {
 	}
 
 	if !env.DryRun {
-		for _, ref := range cleanups {
+		for i := range cleanups {
+			ref := cleanups[i]
```

Comment:
Do you know which linter does this? I am curious why this is better

Reply:
It's `gosec`, see https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable.

It's not "better" per se, but the reference taken here might become invalid shortly, hence it ain't a good idea.

Reply:
That is to say: This isn't necessary in many many cases, using the range clause is fine for most cases actually. Only if you're explicitly taking a reference to the iterator variable are you potentially going to have a bad time.